packages/agoric-cli/template/.agservers/package.json

```
@@ -1,5 +1,5 @@
 {
-  "name": "agoric-servers",
+  "name": "@agoric/dapp-agoric-servers",
```

This doesn't make it too easy for end-developer's "hello world" examples to get accidentally published under the `@agoric/` NPM namespace, does it?

---

No, because these packages are both marked as `"private"`, and also because external (non Agoric org) dapp developers don't have permission to publish to `@agoric/`, so it will fail noisily if they remove the `"private"` and try without renaming the package.